prow/jobs/tools/cmd/command/patch_prowjobs.go

```diff
@@ -94,11 +94,16 @@ func buildProwImages(cmd *cobra.Command, args []string) error {
 	}
 
 	log.Printf("Tags to build:\n %v\n", tagsToBuild)
-	if err = buildImagesWithKaniko(imagesConfig.ImageRepo, tagsToBuild); err != nil {
+	if err = buildImages(tagsToBuild); err != nil {
 		return err
 	}
 	log.Println("Successfully built all images")
 
+	if err = tagAndPushImages(imagesConfig.ImageRepo, tagsToBuild); err != nil {
+		return err
+	}
+	log.Printf("Error")
```

Comment:
Did you mean log.Println("Tagged and pushed images successfully") ?

Reply:
yes